doc/_config.yml

```diff
@@ -18,9 +18,16 @@ include: [_includes]
 # variables
 
 sidebar_tagline: Spoon
-sidebar_version: version 7.0.0
```

Comment:
just for info, can't you define sidebar_version according to spoon_release?

Reply:
that is another option, but I prefer to simply get rid of sidebar_version

Reply:
ok